crates/containerd-shim-wasmedge/src/instance_linux.rs

```diff
@@ -51,14 +50,17 @@ impl Engine for WasmEdgeEngine {
             .initialize(
                 Some(args.iter().map(String::as_str).collect()),
                 Some(envs.iter().map(String::as_str).collect()),
-                Some(vec!["/:/"]),
+                Some(preopens.iter().map(|s| s as &str).collect()),
```

Comment:
I am a bit concerned about enabling preopens by default for wasm modules, see discussions in #266. Is this required to support WasmEdge plugins?

Reply:
we are already preopening the whole root.
I don't think this change is doing what it intended to do, as any path in the array will be taked as a path **inside** the container (as by this point libcontainer will have pivoted the container root).

Reply:
The changes were not essential for supporting the plugin; they were made solely to pass the model file and image file to WASM for reading during the execution of our example use wasinn plugin.

I believe @jprendes 's  opinion is correct; if we are already directly binding the root to WASI, it should meet my requirements. 
> The initial version was implemented as a legacy when preopen was different from the current implementation. At that time, the legacy version of preopen used the root path in oci spec, not the container's root. Later, the readonly flag was added, so all of preopen code was included in this PR. However, I agree that the current architecture meets my requirements.

Reply:
Remove done.